vexim/config/header.php

```diff
@@ -13,9 +15,9 @@
     print     "-- " . $_SESSION['domain'] . " ";
   }
   if (isset($_SESSION['domain_id'])) {
-    if (($domheaderrow['enabled'] == "0") || ($domheaderrow['enabled'] == "f")) {
+    if (($domheaderrow['enabled'] === "0") || ($domheaderrow['enabled'] === "f")) {
```

Comment:
I don't think "f" is actually a valid value here. Integer 0, on the other hand, seems very valid. We should check what is actually returned by the query.

Same for below.